### ISSUE TITLE
Add the possibility to disable registration completely

### ISSUE THREAD

**Issue by marceloschmidt** [RocketChat/Rocket.Chat#224]
Only admins can add/remove users to their infrastructure.

**Comment by engelgabriel** [comment]
Depends on #222

**Comment by graywolf336** [comment]
When adding users, will it be an invite style? As that's how I'm used to it but also just being able to add people without them confirming an invite would help for temporary users. Maybe even ability to add users and make them valid for one week only and expire their login until an admin/person with permissions extends their stay.

**Comment by marceloschmidt** [comment]
Duplicate of #195

**Comment by 42robots** [comment]
Is it possible to disable the "Register new Account" on the login page?

Otherwise, i agree that this must be possible, because right know everyone could enter the chat
if mail confirmation is disabled.
